docs/cli/kops_create_cluster.md

```diff
@@ -26,7 +26,7 @@ kops create cluster
       --master-zones string         Zones in which to run masters (must be an odd number)
       --model string                Models to apply (separate multiple models with commas) (default "config,proto,cloudup")
       --network-cidr string         Set to override the default network CIDR
-      --networking string           Networking mode to use.  kubenet (default), classic, external, cni, kopeio-vxlan, weave, calico. (default "kubenet")
+      --networking string           Networking mode to use.  kubenet (default), classic, external, cni, kopeio-vxlan, weave, flannel, calico. (default "kubenet")
```

Comment:
I've tested the following:

`flannel -> weave` works as expected
`flannel -> weave -> cni` doesnt work as expected(?), weave still running
`flannel -> cni` doesnt work as expected(?), flannel still running
`flannel -> cni -> kubenet` doesnt work as exected(?), flannel is still running

My guess is that moving away from the initial network provider works if its replacing the provider in a similiar way (weave/flannel)